book/src/proposals/tower-lockout-slashing.md

```diff
@@ -26,6 +27,11 @@ prover needs to submit two votes.
 * vote 3: 0, 1, 2, 3
 * vote 4: 0, 1, 2, 5
 
+The second vote skips 3, and therefore allows the vote to be accepted
+on a fork that doesn't include it, such as `0,1,2,4,5`.  This
+violates the lockout rule for 3, since the earliest vote that can
+be accepted that doesn't contain it is 6.
```

Comment:
Do we also need to include the lockouts for each vote, or can we derive lockouts by just looking at the vote histories included in each vote?  For instance,

vote 0: 0, 1, 2, 9
vote 1: 0, 1, 5

vote 0 implies that the lockout on slot 2 is at least 8, which would mean vote 1 is a violation.